test/egeoip_tests.erl

```diff
@@ -59,17 +62,27 @@ egeoip() ->
     {ok, IpAddressLong} = egeoip:ip2long("207.145.216.106"),
     {ok, IpAddressLong} = egeoip:ip2long(<<207,145,216,106>>),
     {ok, R} = egeoip:lookup(IpAddressLong),
+    Region
+        = case application:get_env(egeoip, dbfile) of
+              {ok, city} -> <<"CA">>;
+              _ -> <<>>
+          end,
     #geoip{country_code = "US",
            country_code3 = "USA",
            country_name = "United States",
-           region = <<"CA">>,
+           region = Region,
            _ = _} = R,
```

Comment:
I would suggest always using the [assert macros](http://erlang.org/doc/apps/eunit/chapter.html), as they tend to give more helpful output in case of failure.